### DISCUSSION TITLE
Query regd. capability

### DISCUSSION THREAD

**Discussion by gauravity30** [andrsd/gmsh2exo#23] [Q&A] [answered]
Hii,
Among all the GMSH2exo converters, I found yours to be the latest updated. Does it support conversion of Physical volumes --> Blocks and Physical Surfaces --> Sidesets ?

**Comment by andrsd** [comment, marked as answer]
TLDR; yes. It is why I wrote this small code. The other converters were not doing this. But I was not looking too hard, either.

Detailed answer:
In 3D: physical volumes are converted to blocks and physical surfaces to side sets,
in 2D: physical surfaces become blocks and physical curves become side sets.

I think I need to rewrite the "features" section in README, it is not very clear that the code does this.

Note that only 1st order elements are currently supported, i.e. LINE2, TRI3, TET4 and HEX8. If you wanted 2nd order elements, I would have to add that. This is tracked by issue #15.

Also, I see I don't have discussions active in this project. 

Hope this code will be helpful to you...

**Reply by gauravity30** [reply]
Thanks for the reply ! I see, this might help me. Can you add steps to install the tool along with its dependencies, it is not very clear rn.

**Reply by andrsd** [reply]
Yes. I hope to get to that this weekend. 

I assume you have things like exodusII from SEACAS already installed? SEACAS is a dependency for exodusIIcpp (which wraps the C/Fortran API)

**Reply by andrsd** [reply]
I added build instructions to the [README](https://github.com/andrsd/gmsh2exo?tab=readme-ov-file#how-to-build). Hope this helps you to build the code.
